Formula/daemontools.rb

```diff
@@ -15,16 +16,28 @@ class Daemontools < Formula
 
   def install
     cd "daemontools-#{version}" do
+      inreplace ["package/run", "src/svscanboot.sh"] do |s|
+        s.gsub! "/service", "#{etc}/service"
+      end
+
       system "package/compile"
       bin.install Dir["command/*"]
     end
   end
 
+  def post_install
+    (etc/"service").mkpath
+
+    Pathname.glob("/service/*") do |original|
+      target = "#{etc}/service/#{original.basename}"
+      ln_s original, target unless File.exist?(target)
+    end
+  end
+
   def caveats
     <<~EOS
-      You must create the /service directory before starting svscan:
-        sudo mkdir /service
-        sudo brew services start daemontools
+      Create your service directories under:
```

Comment:
I think `Services are stored in:` might be better.

Reply:
@SMillerDev Done, thanks.